priv/templates/boot.eex

```diff
@@ -613,11 +613,11 @@ case "$1" in
             mv "$REL_DIR/vm.args.bak" "$REL_DIR/vm.args"
         fi
         # Then, backup the packaged configs
-        cp "$REL_DIR/sys.config" "$REL_DIR/sys.config.bak"
-        cp "$REL_DIR/vm.args" "$REL_DIR/vm.args.bak"
+        cp -afv "$REL_DIR/sys.config" "$REL_DIR/sys.config.bak"
```

Comment:
Leaving off the -f seems to fail for me. Output from --debug is:

cp: cannot create regular file '/home/web/appname/releases/version/sys.config': Permission denied

But running the command myself I get:

cp: 'path/to/old/sys.config' and '/home/web/appname/releases/version/sys.config' are the same file

So maybe the correct answer is just to test if they are the same file or not before copying?

My bash-fu is not that strong so I am not 100% sure of the best way to do that. Maybe "cmp -s old new"?